src/app/zap-templates/zcl/data-model/chip/joint-fabric-pki-cluster.xml

```diff
@@ -0,0 +1,89 @@
+    <item name="FailedDclVendorIdValidation" value="0x03"/>
+    <item name="NotAnIcac" value="0x04"/>
+    <item name="BusyAnchorTransfer" value="0x05"/>
+    <item name="IcaCsrSigningFailed" value="0x06"/>
+    <item name="IcaCsrRequestNoUserConsent" value="0x07"/>
+  </enum>
+
+  <enum name="TransferAnchorResponseStatusEnum" type="enum8">
+    <cluster code="0x0753"/>
+    <item name="OK" value="0x00"/>
+    <item name="TransferAnchorStatusDatastoreBusy" value="0x01"/>
+    <item name="TransferAnchorStatusNoUserConsent" value="0x02"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Joint Fabric Pki</name>
+    <code>0x0753</code>
+    <define>JOINT_FABRIC_PKI_CLUSTER</define>
+    <description>Joint Fabric Pki Cluster.</description>
+
+    <command source="client" code="0x00" name="ICACSRRequest" response="ICACSRResponse" optional="false">
+      <description>Client requests Server's ICA CSR.</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x01" name="ICACSRResponse" optional="false">
+      <description>Server requests ICAC to be signed by Client's PKI.</description>
+      <arg name="NOCIssuerCSR" type="octet_string" length="400"/>
+    </command>
+
+    <command source="client" code="0x02" name="AddICAC" response="AddICACResponse" optional="false">
+      <description>Joint Fabric ICAC generated.</description>
+      <arg name="StatusCode" type="ICACSigningRequestStatusEnum"/>
+      <arg name="NOCIssuerCert" type="octet_string" length="400"/>
+      <arg name="NodeId" type="node_id"/>
+      <arg name="FabricId" type="fabric_id"/>
+      <arg name="AdminVendorId" type="vendor_id"/>
+      <arg name="CaseAdminSubject" type="int64u"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x03" name="AddICACResponse" optional="false">
+      <description>Response to AddICAC command.</description>
+      <arg name="StatusCode" type="ICACSigningRequestStatusEnum"/>
+      <arg name="FabricIndex" type="fabric_idx" optional="true"/>
+      <arg name="DebugText" type="char_string" optional="true" length="128"/>
+    </command>
+
+    <command source="client" code="0x04" name="TransferAnchorRequest" response="TransferAnchorResponse" optional="false">
+      <description>JF Admin candidate requests transfer of the Anchor Fabric.</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="server" code="0x05" name="TransferAnchorResponse" optional="false">
+      <description>Transfer Anchor response.</description>
+      <arg name="StatusCode" type="TransferAnchorResponseStatusEnum"/>
+    </command>
+
+    <command source="client" code="0x06" name="TransferAnchorComplete" optional="false">
+      <description>Transfer Anchor complete.</description>
+      <access op="invoke" privilege="administer"/>
+    </command>
+  </cluster>
+</configurator>
```

Comment:
I don't see the Open Joint Commissioning Window Command as described by https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10743